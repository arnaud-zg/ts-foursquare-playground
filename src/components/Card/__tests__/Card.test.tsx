import React from 'react'
import renderer from 'react-test-renderer'
import { Card, CardContent } from '..'

describe('Component/Card', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Card />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Component/CardContent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<CardContent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
