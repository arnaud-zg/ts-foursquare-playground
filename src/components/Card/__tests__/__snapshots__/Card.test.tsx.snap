// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Card renders correctly 1`] = `
<article
  className="flex-1 border-b"
>
  <div
    className="h-full border-l-2 bg-grey-lightest border-indigo"
  />
</article>
`;
